chainntnfs/interface.go

```diff
@@ -241,6 +241,12 @@ type SpendDetail struct {
 	SpendingHeight    int32
 }
 
+// String returns a string representation of SpendDetail.
+func (s *SpendDetail) String() string {
+	return fmt.Sprintf("%v[%d] spending %v at height=%v", s.SpenderTxHash,
+		s.SpenderInputIndex, s.SpentOutPoint, s.SpendingHeight)
```

Comment:
This will show a zero outpoint for script spends.

Reply:
Looks like it will be set based on the `TxIn` when a spend is detected, and it therefore should be non-zero?